docs/performance-cluster.md

```diff
@@ -0,0 +1,95 @@
+
+### **VM/VMI end-to-end latency**
+KubeVirt provides high-level abstractions for users to represent their applications, for example using virtual machine (VM) objects or virtual machine instance (VMI) objects. Where VM are stateful objects that can be stopped and started and VMI represents the basic ephemeral building block of an instance. Note that a VM object can create a VMI object. For the simplicity sake, the CI/CD performance test currently only evaluates the VMI object, but we plan to extend it in the future.
+
+Therefore, the most important metric is the VMI creation latency, which directly impacts the end-user experience.
+The VMI creation latency is measured from the time the object is created to the time the VMI object enters in the Running phase. A running VMI means the libvirt domain has been created and the virtual machine OS is booting. So while knowing whether the OS is fully booted or [whether the network is accessible](https://github.com/kubevirt/kubevirt/pull/5946) are also important metrics, they do not depend on the KubeVirt control plane. Hence, these aspects are not being evaluated in CI/CD performance tests at this time.
+
+
+### **API responsiveness for user-level abstractions**
+Another important metric is the tail latency for KubeVirt API operations. The 99th percentile of all API calls should return in less than 1s.
+
+### **VM/VMI creation rate (throughput)**
+A high creation rate is crucial for scalability, on a large scale system the system throughput will affect the overall performance. Throughput is measured as the number of VMI running per second.
+
+
+## Cluster Configuration
+The cluster is a Kubernetes cluster managed on baremetal nodes in the IBM cloud. For the software layer, the cluster is configured with Kubernetes 1.21.3, where each node has CentOS Linux 8. The Kubernetes cluster consists of 3 master nodes and 3 worker nodes, the etcd cluster runs in the master nodes. 
+
+Regarding the hardware configuration, all nodes are homogeneous, with 2 Intel(R) Xeon(R) Platinum 8260 CPUs @ 2.40GHz where each processor has 24 cores and 2 hyper-threads per core, thus with a total of 96 CPUs. In addition, each node has 128GB of DRAM, 10Gbps network bandwidth, and 1.6T of SCSI disk.
+
+
+## Components
+* Load generator: [perfscale-load-generator](https://github.com/kubevirt/kubevirt/tree/main/tools/perfscale-load-generator), which is a tool aimed at stressing the Kubernetes and KubeVirt control plane by creating several objects (e.g., VM, VMI, and VMIReplicaSet).
+
+* Monitoring stack: prometheus, node-exporter, and grafana. 
+   * Prometheus retains data for up to 3 months.
+   * Various dashboards including a KubeVirt control-plane dashboard.
+
+
+## Exposed services
+
+* deck: Prow UI, available at https://prow.ci.kubevirt.io
```

Comment:
Do you consider it as helpful, if the names "periodic-kubevirt-performance-cluster-scale-density-test" and "periodic-kubevirt-performance-cluster-100-density-test" would be mentioned here again, e.g.

```suggestion
* deck: "periodic-kubevirt-performance-cluster-scale-density-test" and "periodic-kubevirt-performance-cluster-100-density-test"  in Prow UI, available at https://prow.ci.kubevirt.io
```

Reply:
Yes, it is a good point! Thanks